src/Uno.UI.RuntimeTests/Tests/HotReload/Frame/HRApp/Tests/Given_TextBox.cs

```diff
@@ -0,0 +1,54 @@
+﻿using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Uno.UI.RuntimeTests.Tests.HotReload.Frame.Pages;
+
+namespace Uno.UI.RuntimeTests.Tests.HotReload.Frame.HRApp.Tests;
+
+[TestClass]
+[RunsOnUIThread]
+public class Given_TextBox : BaseTestClass
+{
+	public const string SimpleTextChange = " (changed)";
+
+	public const string FirstPageTextBlockOriginalText = "First page";
+	public const string FirstPageTextBlockChangedText = FirstPageTextBlockOriginalText + SimpleTextChange;
+
+	public const string SecondPageTextBlockOriginalText = "Second page";
+	public const string SecondPageTextBlockChangedText = SecondPageTextBlockOriginalText + SimpleTextChange;
+
+
+	/// <summary>
+	/// Checks that when simple change to a XAML element (change Text on TextBlock) is applied to
+	/// the currently visible page, the UI update process will capture and restore property values
+	/// of specific controls (in this case the Text property of a TextBox
+	/// </summary>
+	[TestMethod]
+ 	[Ignore("Not yet working")]
+	public async Task When_Changing_TextBox()
```

Comment:
We'll need to ensure that restoring properties that are commonly bound is not causing issues when two-way is used

Reply:
This is only a test scenario to validate that the extension is working.